tests/kernel/common/prj_hw_random_xoroshiro.conf

```diff
@@ -0,0 +1,8 @@
+CONFIG_ZTEST=y
+CONFIG_RING_BUFFER=y
+CONFIG_RNG=y
+CONFIG_SYS_LOG=y
+CONFIG_ENTROPY_HAS_DRIVER=y
+CONFIG_ENTROPY_GENERATOR=y
+CONFIG_XOROSHIRO_RANDOM_GENERATOR=y
```

Comment:
The Xoroshiro128+ requires an entropy driver -- otherwise, it's going to be seeded with the number of CPU cycles until it is properly initialized (usually an entropy driver not returning an error when asking for a few bytes of entropy to seed the PRNG).